src/main/java/rpgram/RPGram.java

```diff
@@ -136,9 +136,9 @@ public void onUpdateReceived(Update update) {
             }
             case "map":
             case "back": {
-                answer = curPlayer.getMap().viewMapArea(curPlayer.getPos(), curPlayer.getFov());
+                String mapArea = curPlayer.getMap().viewMapArea(curPlayer.getPos(), curPlayer.getFov());
```

Comment:
This line is too long.  I suggest to make it shorter.  For example:
```suggestion
                String mapArea = curPlayer.getMap().viewMapArea(
                        curPlayer.getPos(),
                        curPlayer.getFov()
                    );
```